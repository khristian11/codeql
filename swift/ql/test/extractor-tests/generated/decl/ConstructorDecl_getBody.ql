import codeql.swift.elements
import TestUtils

from ConstructorDecl x
where toBeTested(x)
select x, x.getBody()
