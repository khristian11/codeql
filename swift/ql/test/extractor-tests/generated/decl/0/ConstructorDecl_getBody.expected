| declarations.swift:3:8:3:8 | init | declarations.swift:3:8:3:8 | { ... } |
| declarations.swift:11:7:11:7 | init | declarations.swift:11:7:11:7 | { ... } |
| declarations.swift:44:3:46:3 | init | declarations.swift:44:10:46:3 | { ... } |
| declarations.swift:70:25:70:25 | init | declarations.swift:70:25:70:25 | { ... } |
